website/static/js/profile.js

```diff
@@ -577,6 +609,41 @@ var ListViewModel = function(ContentModel, urls, modes) {
         }
         return true;
     });
+    
+
+    self.institutionsEmpty = ko.computed(function() {
+
+            for (var i=0; i<self.contents().length; i++) {
+                if (self.contents()[i].institutionEmpty()) { 
+                    return true;
+                }
+            }
+ 
+        return false;
+                
+    });
+
+    
+    self.extraFieldsEmpty = ko.computed(function() {
+        var i;
+        if (urls.crud === '/api/v1/settings/jobs/') {
```

Comment:
This is definitely cleaner code.  There seems to be some sequencing issues with implementing it.  (Knockout does not have access to fieldtype when extrafieldsempty is implemented)